resources/js/components/IndexField.vue

```diff
@@ -1,9 +1,17 @@
 <template>
-  <span>{{ Object.keys(field.value).length }} {{ field.name }}(s)</span>
+  <span>{{ indexLabel }}</span>
 </template>
 
 <script>
 export default {
     props: ['resourceName', 'field'],
+
+    computed:{
+        indexLabel: function(){
+            let count = Object.keys(this.field.value).length;
+            let label = (count > 1) ? this.field.pluralLabel : this.field.singularLabel;
+            return `${count} ${label}`;
```

Comment:
Maybe even omit the count if it's just a single item anyway?
```suggestion
            return $count === 1 ? $label : `${count} ${label}`;
```

Reply:
@Anticom I think it might not look even considering we are showing a number for 0 and as well.